Update TFC API Notification Payload to mention null values in payload (#407)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/notification-configurations.mdx

}
```

##### `Send a test` Payload

> This is a sample payload during a test notification. The payload will be void of a `run` or `workspace` context, resulting in null values.

Reviewer: ```suggestion
This is a sample payload you can send to test if notifications are working. The payload does not have a `run` or `workspace` context, resulting in null values.
```
